src/hooks/useCds/translate.js

```diff
@@ -37,7 +37,7 @@ const testCodeResultMapping = [
       },
       {
         text: "High grade squamous intraepithelial lesion, Carcinoma in situ dysplasia",
```

Comment:
There are gaps of text for the code 416033009 (reference: https://ncit.cancer.gov/ncitbrowser/pages/concept_details.jsf?dictionary=SNOMED%20Clinical%20Terms%20US%20Edition&version=2022_03_01&ns=SNOMED%20Clinical%20Terms%20US%20Edition&code=416033009)

`Cervicovaginal cytology: High grade squamous intraepithelial lesion or carcinoma`

Or the text stick to `High grade squamous intraepithelial lesion, Carcinoma in situ dysplasia` as fixed, but `416033009` is the most closed one?

Reply:
The `text` field here is the actual display text that is coming from the pilot partner, whereas the `code` field indicates the SNOMED CT code we are mapping it to. So the `text` field is not necessarily supposed to represent the display value for that specific SNOMED code.

Reply:
Got it.